tracing-appender/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+use chrono::{DateTime, Datelike, TimeZone, Timelike, Utc};
+
+pub mod file_appender;
+
+mod inner;
+mod worker;
+
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+pub enum Rotation {
+    Hourly,
+    Daily,
+    Never,
+}
+
+impl Rotation {
+    fn next_date(self, current_date: &DateTime<Utc>) -> DateTime<Utc> {
```

Comment:
I'm assuming by timestamp you mean the time appended to the filename and not the timestamp printed for each event. This code doesn't change the timestamp format in the latter case.

I see the benefits of using FormatTime but how would we be able to to ensure the log's file name still matches hourly/daily format? I think we'd need to have another formatter that can handle formatting any of ChronoUTC,ChronoLocal,Uptime,Systemtime.

Reply:
Hmm, I think it would be fine to always use UTC timestamps for now, and worry about this later. Adding an API for overriding how filename timestamps are generated shouldn't be a breaking change. Let's save that for a follow-up.